static/app/utils/url/normalizeUrl.tsx

```diff
@@ -18,7 +18,10 @@ const NORMALIZE_PATTERNS: Array<[pattern: RegExp, replacement: string]> = [
     '/settings/$1',
   ],
   [/^\/?join-request\/[^\/]+\/?.*/, '/join-request/'],
-  [/^\/?onboarding\/[^\/]+\/(.*)/, '/onboarding/$1'],
+  [
+    /^\/?onboarding\/(?!setup-docs|select-platform|welcome)[^\/]+\/(.*)/,
+    '/onboarding/$1',
+  ],
```

Comment:
fixed an issue with the regex that caused the router to redirect to `onboarding/` when the parameter was `setup-docs`, `select-platform`, or `welcome`.